library/src/extra/rocsparse_csrgeam_symbolic_impl.cpp

```diff
@@ -0,0 +1,298 @@
+                                                rocsparse_index_base idx_base_A,
+                                                rocsparse_index_base idx_base_B,
+                                                rocsparse_index_base idx_base_C)
+    {
+        rocsparse::csrgeam_fill_symbolic_multipass_device<BLOCKSIZE, WFSIZE>(m,
+                                                                             n,
+                                                                             csr_row_ptr_A,
+                                                                             csr_col_ind_A,
+                                                                             csr_row_ptr_B,
+                                                                             csr_col_ind_B,
+                                                                             csr_row_ptr_C,
+                                                                             csr_col_ind_C,
+                                                                             idx_base_A,
+                                                                             idx_base_B,
+                                                                             idx_base_C);
+    }
+}
+
+namespace rocsparse
+{
+    static rocsparse_status csrgeam_symbolic_quickreturn(rocsparse_handle             handle,
+                                                         rocsparse_operation          trans_A,
+                                                         rocsparse_operation          trans_B,
+                                                         int64_t                      m,
+                                                         int64_t                      n,
+                                                         const rocsparse_mat_descr    descr_A,
+                                                         int64_t                      nnz_A,
+                                                         const void*                  csr_row_ptr_A,
+                                                         const void*                  csr_col_ind_A,
+                                                         const rocsparse_mat_descr    descr_B,
+                                                         int64_t                      nnz_B,
+                                                         const void*                  csr_row_ptr_B,
+                                                         const void*                  csr_col_ind_B,
+                                                         const rocsparse_mat_descr    descr_C,
+                                                         const void*                  csr_row_ptr_C,
+                                                         void*                        csr_col_ind_C,
+                                                         const rocsparse_spgeam_descr descr,
```

Comment:
Not sure why we are not consistent with the place of the spgeam descriptor in the list of parameters.